addons/mail/static/src/model/model_manager.js

```diff
@@ -793,11 +819,19 @@ export class ModelManager {
             }
         }
         this._createdRecords.delete(record);
-        for (const listener of this._listenersObservingLocalId.get(record.localId)) {
-            this._markListenerToNotify(listener);
+        for (const [listener, infoList] of this._listenersObservingLocalId.get(record.localId).entries()) {
+            this._markListenerToNotify(listener, {
+                reason: `_delete: localId - ${Model}`,
```

Comment:
consistency: print Model or Model.modelName everywhere

Suggestion: define toString on Model to return modelName and just use Model.